lib/independent_reserve_client/requests/get_exchange_rate_request.rb

```diff
@@ -0,0 +1,30 @@
+module IndependentReserveClient
+  class GetExchangeRateRequest < BaseRequest
+
+    attribute :primary_currency, String
+    attribute :secondary_currency, String
+
+    private
+
+    def path
+      # url should be /PublicGetMarketSummary?
+      # change it to /PublicGetMarketSummary=
+      # because path is not recognized
+      # replace it back in the default_uri method below
+
+      "/Public/GetMarketSummary=primaryCurrencyCode=:primary_currency" +
```

Comment:
If you want to send params, you can supply `params`. See [api_client_base’s README](https://github.com/imacchiato/api_client_base-ruby/blob/d6bbdfe85e9841b79e8355038e88aeef92e146dd/README.md#requests).

Reply:
When you say "naming convention", should I also follow their method names? For example, `.get_exchange_rate`, I am using `GetMarketSummary` method call, should I instead name it `get_market_summary`, and the response attributes should the same as [documented here](https://www.independentreserve.com/API#GetMarketSummary)?